docs/guides/tutorials/getting_started_v3_api/set_up_data_docs.rst

```diff
@@ -0,0 +1,85 @@
+.. _tutorials__getting_started_v3_api__set_up_data_docs:
+
+How to use Data Docs
+========================
+
+:ref:`Data Docs <reference__core_concepts__data_docs>` translate :ref:`Expectations`, :ref:`Validation Results <reference__core_concepts__validation__expectation_validation_result>`, and other metadata into clean, human-readable documentation. Automatically compiling your data documentation from your data tests in the form of Data Docs guarantees that your documentation will never go stale.
+
+In the previous steps, when you executed the last cell in the Jupyter notebook, Great Expectations used the Expectation Suite you generated to validate the January data batch. It then compiled those validation results to HTML, and opened a browser window with a Data Docs validation results page:
+
+.. figure:: /images/data_docs_taxi_demo01.png
+
+The validation results page shows you the results of using your Expectation Suite to validate a batch of data. In this case, you see the results of validating the ``yellow_tripdata_sample_2019-01`` file. All Expectations were automatically generated using the Profiler functionality, which we will explain below.
+
+If you scroll down, you will see all Expectations that were generated for the ``passenger_count`` column. This includes the Expectation we wanted: **"distinct values must belong to this set: 1, 2, 3, 4, 5, 6"**.
+
+We also see the **observed values** for this batch, which is exactly the numbers 1 through 6 that we expected. This makes sense, since we're developing the Expectation using the January data batch.
+
+.. figure:: /images/data_docs_taxi_demo02.png
+
+**Feel free to click around and explore Data Docs a little more.** You will find two more interesting features:
+
+#. If you click on the *Home* page, you will see a list of all validation runs.
+#. The *Home* page also has a tab for your Expectation Suites, which shows you the Expectations you've created without any validation results.
+
+For now, your static site is built and stored locally. In the last step of the tutorial, we'll explain other options for configuring, hosting and sharing it.
+
+.. _tutorials__getting_started__create_your_first_expectations__what_just_happened:
+
+How did we get those Expectations?
+--------------------------------------
+
+You can create and edit Expectations using several different workflows. Using an automated :ref:`Profiler <Profilers>` as we just did is one of the quickest option to get started with an Expectation Suite.
```

Comment:
The reference was not rendered into a link - not sure if this is just a CI thing.

Reply:
Needs to be lowercase, fixed!